src/stories/typography/Headers.stories.tsx

```diff
@@ -42,6 +42,28 @@ Default.args = {
   useTheme: false,
 };
 
+export const FormattedText: Story<Omit<HeaderProps, 'size'>> = (args) => {
```

Comment:
I think you can re-use the `Default` story 'component' like I did in my draft PR:

https://github.com/VEuPathDB/core-components/pull/39/files#diff-142e234956197d52ff648da0365d00fb979141d334b5593d19f7e660c3c58d48R45-R50